### ISSUE TITLE
Feature: support vsce flags `--allow-missing-repository`and `--allow-star-activation`

### ISSUE THREAD

**Issue by ps610** [felipecrs/semantic-release-vsce#709]
- https://github.com/microsoft/vscode-vsce/issues/700
- https://github.com/microsoft/vscode-vsce/issues/706

**Comment by felipecrs** [comment]
You can configure `vsce` through `package.json`:

https://github.com/microsoft/vscode-vsce#configuration

Thus there should not be a need to implement anything for this, except maybe bumping vsce to the latest version.

**Comment by ps610** [comment]
I am aware of this. I also tested it, but initially only managed to do it using the flags. 
I have now looked in the `vsce` code and my mistake was that I assumed the flags had the same naming scheme in the package.json, but this is not the case.

`--allow-missing-repository` is represented by:
```json
"vsce": {
    "allowMissingRepository": true
  }
```

`--allow-star-activation` is represented by:
```json
"vsce": {
    "allowStarActivation": true
  }
```

This applies obviously to all other option flags (e.g. `--skip-license` => `skipLicense`) too. 
So, my bad, sorry.
